### ISSUE TITLE
Send a notification to the user when the league they signed up for starts.

### ISSUE THREAD

**Issue by 2Pacalypse-** [ShieldBattery/ShieldBattery#967]
Not sure if this is something we want, but it sounds to me like a good use case for our awesome notification system :)

**Comment by tec27** [comment]
Yeah, seems like a good idea. Will probably require some thought/effort as leagues right now don't require any timed events on the server (they just check things as requests come in)
